src/LowLevelMotionController.cpp

```diff
@@ -231,36 +265,120 @@ int main(int argc, char **argv)
         {
             last_time = current_time;
 
+            if(server.isNewGoalAvailable() && !server.isActive())
+            {
+                const iarc7_motion::GroundInteractionGoalConstPtr& goal = server.acceptNewGoal();
+                if ("takeoff" == goal->interaction_type)
+                {
+                    bool success = takeoffController.resetForTakeover(current_time);
+                    if(success)
+                    {
+                        ROS_DEBUG("Transitioning to takeoff mode");
+                        motion_state = MotionState::TAKEOFF;
+                    }
+                    else
+                    {
+                        ROS_ERROR("Failure transitioning to takeoff mode");
+                        server.setAborted();
+                        motion_state = MotionState::VELOCITY_CONTROL;
```

Comment:
I think it should be in this PR because the states are a new thing being introduced here in LLM.  I don't think we should start off in velocity mode.